src/config/dynamic-theme-fixes.config

```
@@ -37691,6 +37691,18 @@ INVERT
 
 ================================
 
+zohranfornyc.com
+
+INVERT
+.bg-background > .mb-4
+
+CSS
+.bg-\[url\(\/static\/images\/clipping\.svg\)\] {
```

Is it a real class name?

---

It's defined in https://www.zohranfornyc.com/_next/static/css/c8ae659f49548637.css as of this writing. This should also work if you wanted to go with a more readable name:

```suggestion
.p-px > .h-2 {
```